minio/templates/network_policy.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: extensions/v1beta1
```

Comment:
Wrap your network policy in condition. We don't alway need network policy, in minikube for example.